tests/jni_api.rs

```diff
@@ -418,6 +418,28 @@ fn convert_byte_array() {
     assert_eq!(dest.unwrap(), src);
 }
 
+#[test]
+fn local_ref_null() {
```

Comment:
Would you please add a test with auto_local to ensure it works correctly with these changes?